spec/fortnox/api/repositories/unit_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require 'fortnox/api'
+require 'fortnox/api/mappers'
+require 'fortnox/api/repositories/unit'
+require 'fortnox/api/repositories/examples/all'
+require 'fortnox/api/repositories/examples/save'
+require 'fortnox/api/repositories/examples/save_with_specially_named_attribute'
+
```

Comment:
I would add a `TODO` in this spec to remember to test the `find` method here later on when we have a test for that.